lib/src/chunk.dart

```diff
@@ -250,58 +250,6 @@ class ChunkBlock {
   ChunkBlock(this.argument);
 }
 
-/// Constants for the cost heuristics used to determine which set of splits is
```

Comment:
I renamed "whitespace.dart" to "constants.dart" and moved this over to that file mainly because it seemed silly to have a file with just one tiny set of constants. Also, having this in "chunk.dart" was always kind of arbitrary.